pkg/jx/cmd/init.go

```diff
@@ -925,3 +925,19 @@ func (o *CommonOptions) GetDomain(client kubernetes.Interface, domain string, pr
 
 	return domain, nil
 }
+
+//NoWhiteSpaceValidator is an input validator for the survey package that disallows any whitespace in the input
+func NoWhiteSpaceValidator() survey.Validator {
```

Comment:
I would be nice if you could move this function in the `util/survey.go`, that can be reused in other parts of the code.

Reply:
that was actually something i was going to ask about this morning :)